### PR TITLE
Update word case

### PR DESCRIPTION
**Pull request by kianmeng**
@Gazler As discussed.

### CODE REVIEW

File: guides/request_lifecycle.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ And this is what the template should look like:
 </div>
 ```
 
-Our messenger appears as `@messenger`. We call "assigns" the values passed from the controller to views. It is a special bit of metaprogrammed syntax which stands in for `assigns.messenger`. The result is much nicer on the eyes and much easier to work with in a template.
+Our messenger appears as `@messenger`. We call "assigns" the values passed from the controller to views. It is a special bit of meta-programmed syntax which stands in for `assigns.messenger`. The result is much nicer on the eyes and much easier to work with in a template.
```
Comment (aaronrenner) [inline guides/request_lifecycle.md:241]: Wikipedia shows metaprogramming as one word without the dash: https://en.wikipedia.org/wiki/Metaprogramming. Would you mind switching this one back?

**Review by aaronrenner — COMMENTED** [review]
@kianmeng Thanks for this! Just one small request...

File: guides/request_lifecycle.md (hunk 1 above)
Comment (kianmeng) [reply guides/request_lifecycle.md]: > Wikipedia shows metaprogramming as one word without the dash: https://en.wikipedia.org/wiki/Metaprogramming. Would you mind switching this one back?

Done.

**Comment by Gazler** [comment]
Thanks <3

**Pull request merged**